minecraft_servers/marsmc/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "server_name": "marsmc",
+  "nice_name": "MarsMC",
+  "direct_ip": "marsmc.net",
+   "server_wildcards":[
+     "play.marsmc.net",
+     "premium.marsmc.net",

```

Comment:
You doesn't need these because of the generic wildcard entry below

Reply:
Done ✅